### ISSUE TITLE
[Feature Request] Make the site multilingual

### ISSUE THREAD

**Issue by yuhao7025** [FOSSBilling/fossbilling.org#21]
I remember that the previous website could be translated, and after changing to the new website, all translations disappeared.

Is it possible to make the website support multiple languages ​​to increase the usage of more people?

**Comment by BelleNottelling** [comment]
it looks like it shouldn't be too difficult to add localization:
https://nextra.site/docs/guide/i18n#use-add-locale-to-filenames

**Comment by John-S4** [comment]
Yes, but in my opinion translations of the website should only be allowed to go live if they are complete translations.

There is also the problem that, while the software can be translated once basically, the website is going to change more often. There will be new changelogs, news, some blog posts in the future. There is no guarantee that if someone translates the site today, they are going to come back when the next release comes out and translate the changelog for us. 

Those aren't reasons not to make it multilingual, just things that I think we have to find solutions to if we are going to.

**Comment by BelleNottelling** [comment]
> There is also the problem that, while the software can be translated once basically, the website is going to change more often. There will be new changelogs, news, some blog posts in the future. There is no guarantee that if someone translates the site today, they are going to come back when the next release comes out and translate the changelog for us.

As usual, you have some great points. I don't know what the best way to handle that would be. Especially since now that I am thinking about it, if a translated page doesn't get updates, it won't at all display new information. With the actual software, it'll default to the original (English) string, but I don't think that'll be possible in this situation. It really is "all or nothing"

**Comment by jaapmarcus** [comment]
> Yes, but in my opinion translations of the website should only be allowed to go live if they are complete translations.

I don't mind if the Change log is not translated for example or even the blog /news page but the major issue is how ever information get out of date way the often as it is active development...

**Comment by admdly** [comment]
Duplicates #9.
